include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -237,6 +269,8 @@ typedef struct _oe_sgx_enclave_properties
     ALLOW_DEBUG,                                                          \
     CAPTURE_PF_GP_EXCEPTIONS,                                             \
     REQUIRE_KSS,                                                          \
+    CREATE_ZERO_BASE_ENCLAVE,                                             \
```

Comment:
This indicates the feature is SGX2 specific while it's not the case. Any idea? @anakrish

Reply:
Can we re-name the macro, such that it does not tie to the SGX arch version?

Reply:
e.g., `OE_SET_ENCLAVE_SGX_V2`

Reply:
`OE_SET_ENCLAVE_SGX2` is to be used for creating SGX2 enclaves, as described here: https://github.com/openenclave/openenclave/blob/872051dec43edc47dada58df90305891aee98bfb/docs/GettingStartedDocs/buildandsign.md.

We need to also add the `CREATE_ZERO_BASE_ENCLAVE` parameter to `OE_SET_ENCLAVE_SGX` macro. This would result in making single-line changes to all our tests. This can be done in another PR.

Alternatively we could have separate macros for zero-based enclaves:
`OE_SET_ZERO_BASE_ENCLAVE_SGX` and `OE_SET_ZERO_BASE_ENCLAVE_SGX2`.

Reply:
Created issue #4136 to address this.